cli/src/commands/git.rs

```diff
@@ -537,7 +537,7 @@ fn cmd_git_fetch(
             GitFetchError::InternalGitError(err) => map_git_error(err),
             _ => user_error(err),
         })?;
-        print_git_import_stats(ui, &stats.import_stats)?;
+        print_git_import_stats(ui, &stats.import_stats, Some(tx.repo().base_repo()))?;
```

Comment:
Pass `tx.repo()`.

`tx.repo().base_repo()` doesn't know the change applied by `git::fetch()`.

Reply:
resolved